index.js

```diff
@@ -0,0 +1,142 @@
+'use strict'
+
+if (!WebAssembly) {
+    throw new Error('WebAssembly not supported here.');
+}
+
+var loadModule = require('./generated/argon2.js');
```

Comment:
Only rec I have is naming this something like "emscriptenWasmRunner", or something slightly more descriptive.